src/pages/home/report/ReportActionItem.tsx

```diff
@@ -580,6 +581,9 @@ function ReportActionItem({
         } else if (ReportActionsUtils.isOldDotReportAction(action)) {
             // This handles all historical actions from OldDot that we just want to display the message text
             children = <ReportActionItemBasicMessage message={ReportActionsUtils.getMessageOfOldDotReportAction(action)} />;
+        } else if (action.actionName === CONST.REPORT.ACTIONS.TYPE.SUBMITTED) {
+            const formattedAmount = CurrencyUtils.convertToDisplayString(Math.abs(report?.total ?? 0), report.currency);
```

Comment:
Added `report?.total ?? 0` to pass the ts check, but `total` will be always present if the report action is `submitted`, since that action only exists in expense reports.